frontend/packages/ux-editor/src/converters/simpleComponentConverters/externalSimpleComponentToInternal.ts

```diff
@@ -1,17 +1,37 @@
 import type { FormComponent } from '../../types/FormComponent';
 import { formItemConfigs } from '../../data/formItemConfig';
 import type { ExternalSimpleComponent } from '../../types/ExternalSimpleComponent';
+import { convertDataBindingToInternalFormat } from '../../utils/dataModelUtils';
+import type { IDataModelBindings, IDataModelBindingsKeyValueExplicit } from '../../types/global';
 
 export const externalSimpleComponentToInternal = (
   externalComponent: ExternalSimpleComponent,
   pageIndex: number | null,
 ): FormComponent => {
   const formItemConfig = formItemConfigs[externalComponent.type];
   const propertyPath = formItemConfig?.propertyPath;
+
+  const explicitBindings = convertAllDatamodelBindings(externalComponent);
+
   return {
     ...(propertyPath ? { propertyPath } : {}),
     ...externalComponent,
+    dataModelBindings: explicitBindings,
     itemType: 'COMPONENT',
     pageIndex,
-  };
+  } as FormComponent;
 };
+
+function convertAllDatamodelBindings(
+  externalComponent: ExternalSimpleComponent,
+): IDataModelBindingsKeyValueExplicit {
+  return externalComponent.dataModelBindings
+    ? Object.entries(externalComponent.dataModelBindings).reduce(
+        (acc, [key, value]) => ({
+          ...acc,
+          [key]: convertDataBindingToInternalFormat(value as IDataModelBindings),
+        }),
+        {},
+      )
+    : undefined;
+}
```

Comment:
```suggestion
function convertAllDatamodelBindings(
   bindings?: IDataModelBindingsKeyValue
): IDataModelBindingsKeyValueExplicit | undefined {
  if(!bindings) return undefined;
  
  return Object.entries(bindings).reduce(
        (acc, [key, value]) => ({
          ...acc,
          [key]: convertDataBindingToInternalFormat(value as IDataModelBindings),
        }),
        {},
      )
}
```
Since this function only use the dataModelBindings property of the component that is passed as argument, it can be a bit cleaner to just the send that property instead of the whole component. What you think?